<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialog</class>
 <widget class="QWidget" name="FindReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find and Replace</string>
  </property>
  <widget class="QLineEdit" name="findText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="findLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Find:</string>
   </property>
  </widget>
  <widget class="QLabel" name="replaceLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="replaceText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="findButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Find All</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="caseSensitive">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Case Sensitive</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceAll">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findNext">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Next</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
